app/jobs/event_notification_job.rb

```diff
@@ -0,0 +1,20 @@
+class EventNotificationJob < ApplicationJob
+  def self.perform
+    presenter = {
+      first_name: 'Dennis',
+      last_name: 'Marchand',
+      email: 'dennis@urbantherapy.ca'
+    }
+
+    # Event.requiring_notification.each do |event|
```

Comment:
With @gregdaynes stuff being merged I think this is ready to code now so instead of leaving this stubbed I'll just finish it.